satellite/payments/service.go

```diff
@@ -58,8 +58,8 @@ type PaymentMethod struct {
 // CreateProjectInvoiceParams contains info needed to create project invoice
 type CreateProjectInvoiceParams struct {
 	ProjectName     string
-	CustomerID      string
-	PaymentMethodID string
+	CustomerID      []byte
+	PaymentMethodID []byte
```

Comment:
Is there a specific reason or need why we move to an byte array?

Reply:
`stripe` has `id` as `string`, but you don't know what the type will be for other payment services. With bytes you can store whatever you need

Reply:
Maybe let's make a separate type for it  rather than using `[]byte`, e.g. `type PaymetMethodID []byte`.

Reply:
But I guess, current isn't that bad either.